docs/services/gcp-services-guide.md

```diff
@@ -472,21 +383,21 @@ variables. It should have been populated with our MySQL connection details after
 the claim became `Bound`.
 
 > Binding status tells you whether your resource has been provisioned and ready
-to use. Crossplane binds the actual resource to the claim via changing  the
+to use. Crossplane binds the actual resource to the claim via changing the
 readiness condition to `Bound`. This happens only when the resource is ready
 to be consumed.
 
 * Check to make sure `wordpressmysql` exists and is populated:
 
 *Command*
 ```bash
-kubectl -n app-project1-dev describe secret wordpressmysql
+kubectl describe secret wordpressmysql
 ```
 
 *Output*
 ```bash
 Name:         wordpressmysql
-Namespace:    app-project1-dev
+Namespace:    default
 Labels:       <none>
 Annotations:  <none>
```

Comment:
Consistency nitpick: this will have the propagation annotations included in the AWS guide. Might be nice to either include or exclude them consistently.